lib/patrol.js

```diff
@@ -82,7 +71,7 @@ const patrol = function (stateDescriptor) {
       const fetcher = findClosestFetcher(dataStore, path);
 
       if (fetcher) {
-        fetcher(setCh, path, convertToNative(getStateByPath(currentState, path)));
+        fetcher(rootCursor.refine(path), rootCursor);
```

Comment:
not sure if I understand this change. why do we need the rootCursor if we're already refining ourselves a sub-cursor? 

and since we don't have a `set` channel now, how do the changes get propagated?